txbits/conf/evolutions/default/2.sql

```diff
@@ -258,12 +258,21 @@ begin
     update balances set hold = hold - new_amount
     where currency = ask.base and user_id = ask.user_id;;
 
-    update balances set hold = hold - new_amount * new_price
-    where currency = bid.counter and user_id = bid.user_id;;
+    if bid.price >  new_price then
```

Comment:
Thanks for catching this! Since `bid.price` must always be >= `new_price` (otherwise there would be no match), we could simply replace `new_price` with `bid.price` for these two queries. It would be nice to have a unit test for this.

@vikstrous What do you think about modifying the existing evolution? If this code is running in production, we'll need to make new evolutions only so they will be non-destructive. Perhaps we'll need to run evolutions and commit a schema pg_dump with new evolutions to be able to diff and review them at that point.